app/org/sagebionetworks/bridge/config/BridgeSpringConfig.java

```diff
@@ -315,6 +315,12 @@ public DynamoDBMapper uploadSchemaDdbMapper(AmazonDynamoDB client) {
     public DynamoDBMapper taskDdbMapper(AmazonDynamoDB client) {
         return DynamoUtils.getMapper(DynamoTask.class, client);
     }
+    
+    @Bean(name = "taskIndex")
+    @Autowired
+    public DynamoIndexHelper taskIndex(AmazonDynamoDB client) {
+        return DynamoUtils.getDynamoIndexHelper(DynamoTask.class, "hashKey-runKey-index", client);
+    }
```

Comment:
FYI, you'll need to pull the latest version from the repo. This should now be DynamoIndexHelper.create().